src/Migration/Reader/Map.php

```diff
@@ -247,8 +245,6 @@ public function getFieldMap($document, $field, $type)
             return $this->fieldsMap[$key];
         }
 
-        $this->validateType($type);
```

Comment:
Validation can be omitted because it will be performed in the `isFieldIgnored()` method